### PR TITLE
feat: authentication functionality

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Changes

- login 
- signup
- logout
image

### CODE REVIEW

File: src/pages/Login/Login.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    border-radius: 0;
+}
+
+input[type="password"] {
+    width: 100%;
+    padding: 10px;
+    margin: 8px 0;
+    border: 1px solid;
+    border-radius: 0;
+}
```
Comment (vikrantkode) [inline src/pages/Login/Login.css:42]: avoid writing styling twice

**Review by vikrantkode — APPROVED** [review]
- Functionality is working fine as expected
- you can merge and move ahead with it.

File: src/pages/Login/Login.css (hunk 1 above)
Comment (sharmayatendra) [reply src/pages/Login/Login.css:42]: these are for different inputs

File: src/context/auth-context.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const AuthContext = createContext();
+
+const AuthProvider = ({children}) => {
+    const tokenDetails = localStorage.getItem('token') || ''
+    const [state,dispatch] = useReducer(authReducer,{
+        user:"" , token: tokenDetails , isAuth: !!tokenDetails
+    })
+
+    console.log(state.isAuth , state.token);
```
Comment (dusariudaykumar) [inline src/context/auth-context.js]: remove ```console.log```

**Review by dusariudaykumar — APPROVED** [review]
The login is not working other than that everything looks fine

File: src/context/auth-context.js (hunk 2 above)
Comment (sharmayatendra) [reply src/context/auth-context.js]: will remove

**Pull request merged**